src/environments/environment.prod.ts

```diff
@@ -1,3 +1,13 @@
 export const environment = {
   production: true,
+  firebaseConfig: {
+    apiKey: 'AIzaSyBsQvHvICR-SXu_6vjdeYZaI8RbSGsSdQE',
```

Comment:
committing API keys to github is bad - typically you'd want to change this API key since it's been pushed.

https://docs.github.com/en/github/authenticating-to-github/removing-sensitive-data-from-a-repository

use dotenv for this kind of thing